uk.co.compucorp.civicrm.appraisals/views/appraisal-cycle.html

```diff
@@ -62,202 +62,951 @@ <h2 class="panel-title pull-left panel--appraisal-cycle-header__title">
+                                                                Region:
+                                                            </label>
+                                                            <div class="col-sm-9">
+                                                                <select class="form-control" id="region">
+                                                                    <option value="">---</option>
+                                                                </select>
+                                                            </div>
+                                                        </div>
+                                                    </div>
+                                                    <div class="col-sm-6">
+                                                        <div class="form-group">
+                                                            <label for="status" class="control-label col-sm-3">
+                                                                Status:
+                                                            </label>
+                                                            <div class="col-sm-9">
+                                                                <select class="form-control" id="status">
+                                                                    <option value="">---</option>
+                                                                </select>
+                                                            </div>
+                                                        </div>
+                                                    </div>
+                                                </div>
+                                                <hr>
+                                                <div class="row">
+                                                    <div class="col-sm-4">
+                                                        <div class="form-group form-group-sm">
+                                                            <label for="self-due-from" class="control-label col-sm-7">
+                                                                Self Appraisal Due From:
+                                                            </label>
+                                                            <div class="col-sm-5">
+                                                                <div class="input-group input-group-sm input-group-unstyled">
+                                                                    <input
+                                                                        type="text"
+                                                                        class="form-control"
+                                                                        id="self-due-from"
+                                                                        datepicker-popup="dd/MM/yyyy"
```

Comment:
Sure, but this task is just for creating a static version, the controllers i wrote were just for showing the modals